src/main/java/executor/command/CommandAddIncomeReceipt.java

```diff
@@ -0,0 +1,102 @@
+package executor.command;
+
+import executor.task.TaskList;
+import interpreter.Parser;
+import ui.IncomeReceipt;
+import ui.Ui;
+import ui.Wallet;
+
+import java.util.ArrayList;
+import java.util.Date;
+
+public class CommandAddIncomeReceipt extends Command {
+
+    private String userInput;
+    private Double income;
+    private Date date;
+    private ArrayList<String> tags;
+    private String description;
+
+    /**
+     * Constructor for CommandAddIncomeReceipt subCommand Class.
+     * @param userInput The user input from the CLI
+     */
+    public CommandAddIncomeReceipt(String userInput) {
+        this.commandType = CommandType.IN;
+        this.userInput = userInput;
+        this.description = "You can add a new income receipt in format of 'In $5.00 /tags tag'.";
```

Comment:
This line seems too long. Maybe create a private static final string that stores this default description and assign it inside the constructor.